### ISSUE TITLE
docker-base build is failing since bumping alpine to 3.20

### ISSUE THREAD

**Issue by matifali** [coder/coder#13407]
1. https://github.com/coder/coder/actions/runs/9273991228/job/25515328829
2. https://github.com/coder/coder/actions/runs/9283895251

Started after #13375

**Comment by dannykopping** [comment]
Should we revert https://github.com/coder/coder/pull/13375 in the interim?

**Comment by johnstcn** [comment]
> Should we revert #13375 in the interim?

https://github.com/coder/coder/pull/13410

**Comment by dannykopping** [comment]
```diff
diff --git a/scripts/Dockerfile.base b/scripts/Dockerfile.base
index afa00dd4c..9537c9ca1 100644
--- a/scripts/Dockerfile.base
+++ b/scripts/Dockerfile.base
@@ -10,7 +10,7 @@ RUN apk add --no-cache \
                curl \
                wget \
                bash \
-               git=2.43.4-r0 \
+               git=2.45.1-r0 \
```

fixes the build AFAICS.

**Comment by dannykopping** [comment]
https://github.com/coder/coder/pull/13411

**Comment by dannykopping** [comment]
The issue was auto-closed accidentally by #13411, but it is indeed fixed:

https://github.com/coder/coder/actions/runs/9299829880/job/25594587488
